app/tests/unit/forms/email/emailService.spec.js

```diff
@@ -1,116 +1,129 @@
+  it('should call chesService to send an email with type sendStatusAssigned', () => {
     chesService.merge = jest.fn().mockReturnValue('sent');
+    emailService._mergeEmailTemplate = jest.fn().mockReturnValue('merged');
+
+    const result = emailService._sendEmailTemplate(
+      'sendStatusAssigned',
+      configData,
+      submission,
+      referer
+    );
 
-    const result = await emailService._sendSubmissionConfirmation(form, submission, body, referer);
     expect(result).toBeTruthy();
     expect(chesService.merge).toHaveBeenCalledTimes(1);
+    mockedReadFileSync.mockRestore();
   });
 
-  it('should not call chesService if the flag is false', async () => {
-    form.showSubmissionConfirmation = false;
+  it('should call chesService to send an email with type sendSubmissionConfirmation', () => {
     chesService.merge = jest.fn().mockReturnValue('sent');
+    emailService._mergeEmailTemplate = jest.fn().mockReturnValue('merged');
 
-    await emailService._sendSubmissionConfirmation(form, submission, body, referer);
-    expect(chesService.merge).toHaveBeenCalledTimes(0);
-  });
-});
-
-describe('_sendSubmissionReceived', () => {
-  const form = {
-    submissionReceivedEmails: ['a@c.com', 'd@f.com'],
-    name: 'xyz',
-    id: '456'
-  };
-  const submission = {
-    confirmationId: 'abc',
-    id: '123'
-  };
-
-  it('should call chesService to send an email', async () => {
-    chesService.merge = jest.fn().mockReturnValue('sent');
+    const result = emailService._sendEmailTemplate(
+      'sendSubmissionConfirmation',
+      configData,
+      submission,
+      referer
+    );
 
-    const result = await emailService._sendSubmissionReceived(form, submission, referer);
     expect(result).toBeTruthy();
     expect(chesService.merge).toHaveBeenCalledTimes(1);
+    mockedReadFileSync.mockRestore();
   });
 
-  it('should not call chesService if there are no emails', async () => {
-    form.submissionReceivedEmails = undefined;
+  it('should call chesService to send an email with type sendSubmissionReceived', () => {
     chesService.merge = jest.fn().mockReturnValue('sent');
+    emailService._mergeEmailTemplate = jest.fn().mockReturnValue('merged');
 
-    await emailService._sendSubmissionReceived(form, submission, referer);
-    expect(chesService.merge).toHaveBeenCalledTimes(0);
-  });
+    const result = emailService._sendEmailTemplate(
+      'sendSubmissionReceived',
+      configData,
+      submission,
+      referer
+    );
 
-  it('should not call chesService if there is a blank array', async () => {
-    form.submissionReceivedEmails = [];
-    chesService.merge = jest.fn().mockReturnValue('sent');
-
-    await emailService._sendSubmissionReceived(form, submission, referer);
-    expect(chesService.merge).toHaveBeenCalledTimes(0);
+    expect(result).toBeTruthy();
+    expect(chesService.merge).toHaveBeenCalledTimes(1);
+    mockedReadFileSync.mockRestore();
```

Comment:
You'll want to move `mockedReadFileSync.mockRestore();` into an afterEach section as it applies to all tests within this describe:
```js
afterEach(() => {
   mockedReadFileSync.mockRestore();
});
```